src/app/dialogs/dialog-loan-pay/dialog-loan-pay.component.scss

```diff
@@ -0,0 +1,73 @@
+@import '../../../scss/_variables.scss'; //variables
+
+.mat-dialog-container {
+  height: 210px;
+}
+.dialog-padding{
+  padding: 30px;
+}
+.navbar-container {
+  height: 70px;
+  position: relative;
+  top: 0;
+  left: 0;
+  right: 0;
+  background-color: #333333;
+  .creator-container {
+    .creator-title{
+      width: 100%;
+      position: absolute;
+      top: 50%;
+      left: 50%;
+      color: $fifth;
+      font-size: 22px;
+      text-align: center;
+      padding: 0 15px;
+      @extend %font-black;
+      @include transform(translate(-50%, -50%));
+    }
+  }
+}
+.currency-logo{
+  width: 35px;
+  display: inline-block;
+  margin-right: 5px;
+  margin-left: 10px;
+  margin-top: -5px;
+}
+.rcn-balance{
+  font-size: 22px;
+  @extend %font-bold;
+}
+.half-logo{
+  position: relative; left: 50%; @include transform(translate(-50%, 0%));
+  max-width: 200px;
+}
+
+.-center-top{
+  position: sticky; top: 50%; @include transform(translate(0%, -50%));
+}
+
+.transfer-width{
```

Comment:
This class was on another component and its not used on dialog-loan-pay.component.html. Please check the code before making a copy paste of other components @Deivbid